include/swift/Basic/LangOptions.h

```diff
@@ -325,6 +325,10 @@ namespace swift {
     /// disabled because it is not complete.
     bool EnableCXXInterop = false;
 
+    /// Enable C interop importing bounds safety and lifetime attributes to
+    /// generate Swift wrappers with safe pointer types (experimental).
+    bool EnableBoundsSafetyInterop = false;
```

Comment:
I would prefer that we add a new experimental feature rather than a separate LangOption. I'm a little nervous about putting this under `LifetimeDependence`, because `LifetimeDependence` is going to get reviewed and enabled-by-default before we're ready to start performing this transform.

Reply:
What is the difference between a language feature and and a flag?